test/support/capybara_helpers.rb

```diff
@@ -0,0 +1,13 @@
+module CapybaraHelpers
```

Comment:
This helper is for us to be able to correctly produce links that point to our capybara server. This is needed when testing the creation of iteration as to do it is produced dynamically within the serializer.

Reply:
I would add this exact comment as a code comment

Reply:
I've done this.